compose/desktop/desktop/samples/src/jvmMain/kotlin/androidx/compose/desktop/examples/layout/Main.jvm.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.desktop.examples.layout
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.awt.ComposeDialog
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.launchApplication
+import java.awt.Dimension
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.delay
+
+fun main() {
+    MainScope().launchApplication {
```

Comment:
Because I want to check this line of code https://github.com/JetBrains/androidx/blob/e529c7e79bee073a21bc789f2dc5b10b60e4dddd/compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/UndecoratedWindowResizer.desktop.kt#L69

It is available only with custom creation of ComposeDialog with flags isResizable and isUndecorated:
ComposeDialog().apply {
            isResizable = true
            isUndecorated = true
        }

Reply:
> It is available only with custom creation of ComposeDialog

`Dialog` calls the same code under the hood, and creates UndecoratedWindowResizer

Reply:
`Window` also calls this code

Reply:
Done